Admin/AbstractAdmin.php

```diff
@@ -2479,13 +2477,13 @@ public function getSecurityHandler()
      */
     public function isGranted($name, $object = null)
     {
-        $key = md5(json_encode($name).($object ? '/'.spl_object_hash($object) : ''));
-
-        if (!array_key_exists($key, $this->cacheIsGranted)) {
-            $this->cacheIsGranted[$key] = $this->securityHandler->isGranted($this, $name, $object ?: $this);
-        }
+        @trigger_error(
+            'The '.__METHOD__.' method is deprecated since version 3.12 and will be removed in 4.0.'.
+            ' Use Sonata\AdminBundle\Security\Handler\SecurityHandlerInterface::isGranted instead.',
```

Comment:
This method is also use from twig quite often.

We should tell the twig function replacement on this message too.